### DISCUSSION TITLE
I noticed that you also have an edge gpt, can a question be answered by edge gpt and chatgpt together, similar to aggregation aigc[Feature Request]:

### DISCUSSION THREAD

**Discussion by cypggs** [acheong08/ChatGPT#941] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

I noticed that you also have an edge gpt, can a question be answered by edge gpt and chatgpt together, similar to aggregation aigc

eg:
you: How to realize wealth freedom？

Chatgpt:                                                                             Edgegpt:

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/3694954/221748076-08f6b1f6-ad99-4008-8e85-57b27788ea22.png">

### Proposed workflow

1. Go to ....
2. Press ....
3. ...

**Comment by acheong08** [comment]
I'm working towards my own version of EdgeGPT based on Davinci and DDG
